_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -4227,9 +4227,9 @@
 /obj/structure/rack,
 /obj/effect/decal/cleanable/dirt,
 /obj/item/storage/box/donkpockets,
-/obj/item/kitchen/knife/butcher,
```

Comment:
this is something that you can just print from a hacked autolathe, and which is equivalent in melee force to a circular saw, which can be printed from a not-hacked autolathe

Reply:
yes

Reply:
okay then why are you removing it for being too strong